docs: document project-level noindex setting (#6152)
mintlify/docs: optimize/seo.mdx

  }
```

### Disable indexing for the entire project

> To hide your entire deployment from search engines while keeping it publicly accessible, enable **Don't index project** in your dashboard under **Settings > Deployment > Add-ons > Visibility**.

Reviewer: ```suggestion
To hide your entire deployment from search engines while keeping it publicly accessible, enable **Don't index project** on the [Add-ons](https://app.mintlify.com/settings/deployment/addons) page of your dashboard.
```

When enabled, Mintlify:

- Renders every page with `noindex, nofollow` robots meta tags.
- Returns an empty `sitemap.xml`.
- Returns `404` for `llms.txt` and `llms-full.txt`.
- Clears your search index so pages no longer appear in internal search, the [AI assistant](/assistant/index), or the [MCP server](/ai/model-context-protocol).
- Serves a site-wide disallow `robots.txt`:

  ```txt
  User-agent: *
  Content-Signal: ai-train=no, search=no, ai-input=no
  Disallow: /

>   ```

Reviewer: ```suggestion
When enabled, your site:

- Renders every page with `noindex, nofollow` robots meta tags.
- Returns an empty `sitemap.xml`.
- Returns `404` for `llms.txt` and `llms-full.txt`.
- Clears your search index so pages no longer appear in internal search, the [assistant](/assistant/index), or your [search MCP server](/ai/model-context-protocol).
- Serves a site-wide disallow `robots.txt`:

  ```txt
  User-agent: *
  Content-Signal: ai-train=no, search=no, ai-input=no
  Disallow: /
  ```
```

> Use this when you want to keep a deployment online for internal review, staging, or limited sharing, but do not want it to appear in public search results or be used as training or retrieval data for AI tools.

Reviewer: ```suggestion
Disable indexing when you want to keep a deployment online for internal review, staging, or limited sharing, but do not want it to appear in public search results or for AI tools to use it as training or retrieval data.
```

<Note>

>   Toggling this setting triggers a redeployment of your site. Changes take effect after the redeployment completes.

Reviewer: ```suggestion
  Enabling or disabling indexing redeploys your site. Changes to indexing take effect after the redeployment completes.
```

</Note>

This setting works alongside other indexing controls:

> - **Per-page `noindex`** — Page-level `noindex: true` in frontmatter continues to apply when the project-level setting is off. The project-level setting overrides per-page settings when enabled, so every page is treated as `noindex`.

Reviewer: ```suggestion
- **Per-page `noindex`**: Page-level `noindex: true` in frontmatter continues to apply when the project-level setting is off. Project-level settings override per-page settings when enabled.
```

> - **Custom `robots.txt`** — If you have a [custom `robots.txt`](#custom-sitemaps-and-robotstxt-files) at the root of your project, Mintlify serves it unchanged. The project-level setting does not replace a custom `robots.txt`, so update or remove your custom file separately if you need crawler rules to match.

Reviewer: ```suggestion
- **Custom `robots.txt`**: If you have a [custom `robots.txt`](#custom-sitemaps-and-robotstxt-files) at the root of your project, Mintlify serves it unchanged.  Project-level settings do not replace custom `robots.txt` files, so you must update or remove custom files for crawler rules to match.
```
